src/plugins/kibana_overview/public/components/overview/overview.test.mocks.ts

```diff
@@ -40,3 +58,11 @@ jest.doMock('@kbn/kibana-react-plugin/public', () => ({
 jest.doMock('../../lib/ui_metric', () => ({
   trackUiMetric: jest.fn(),
 }));
+
+export const mockDataViewEditor: DataViewEditorStart = {
+  openEditor: jest.fn(),
+  IndexPatternEditorComponent: jest.fn(),
+  userPermissions: {
+    editDataView: jest.fn(),
+  },
+};
```

Comment:
NIT: I would expect such mock to be exposed from the `data-view-editor` plugin itself?

Reply:
There is one there, but I assumed there was a reason why it wasn't exported.

Reply:
If you look right, it IS exported 😅 Changed this now.